components/ProductMatrix.vue

```diff
@@ -32,37 +36,88 @@ export default Vue.extend({
   },
   data() {
     return {
-      error: '',
+      cardWidth: 152,
```

Comment:
Is there a clever way to show that this is a temporary number that will be recalculated later? Like a function with a default value or something. Hm.

Reply:
What about `cardWidth: initialCardWidth`?